lib/rules/selector-type-no-unknown/__tests__/index.js

```diff
@@ -85,6 +85,10 @@ testRule(rule, {
 			code: 'circle {}',
 			description: 'svg tags',
 		},
+		{
+			code: '.test foreignObject { color: white }',
```

Comment:
Sorry, didn't spot it right away:

```suggestion
			code: 'foreignObject { }',
```

It's for consistency with other test cases. Could you also adjust description to `case-sensitive svg tags`, please?